test/sequential/test-stream-writable-clear-buffer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+const Stream = require('stream');
+//this test ensures that the _writeableState.bufferedRequestCount and
```

Comment:
Nit - would you be so kind upper case the first letter and also add a whitespace after `//`?

Reply:
Sure no problem, would there be any other style changes you'd think might be appropriate? If not I'll just commit with that one change and squash if required/requested.

Reply:
In the comment below, please also add a whitespace on each line after `//`. Besides that everything is LGTM.